mongodb/src/test/scala/docs/scaladsl/MongoSinkSpec.scala

```diff
@@ -36,7 +38,10 @@ class MongoSinkSpec
   case class Number(_id: Int)
   case class DomainObject(_id: Int, firstProperty: String, secondProperty: String)
 
-  val codecRegistry = fromRegistries(fromProviders(classOf[Number], classOf[DomainObject]), DEFAULT_CODEC_REGISTRY)
+  val codecRegistry =
+    fromRegistries(fromProviders(classOf[Number], classOf[DomainObject]), DEFAULT_CODEC_REGISTRY): @nowarn(
+      "msg=match may not be exhaustive"
+    )
```

Comment:
No pattern matching going on here?

Reply:
Very hidden indeed:
```
[warn] /Users/enno/dev/alpakka/mongodb/src/test/scala/docs/scaladsl/MongoSinkSpec.scala:40:41: match may not be exhaustive.
[warn] It would fail on the following input: (x: String forSome x not in "Number")
[warn]     fromRegistries(fromProviders(classOf[Number], classOf[DomainObject]), DEFAULT_CODEC_REGISTRY)
[warn]                                         ^
[warn] /Users/enno/dev/alpakka/mongodb/src/test/scala/docs/scaladsl/MongoSinkSpec.scala:40:58: match may not be exhaustive.
[warn] It would fail on the following input: (x: String forSome x not in "DomainObject")
[warn]     fromRegistries(fromProviders(classOf[Number], classOf[DomainObject]), DEFAULT_CODEC_REGISTRY)
```